crates/shim-runc/build.rs

```diff
@@ -0,0 +1,44 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+use std::fs;
+use std::path::Path;
+use ttrpc_codegen::{Codegen, ProtobufCustomize};
+
+fn main() {
```

Comment:
How about applying the same trick as https://github.com/containerd/rust-extensions/blob/main/crates/shim-protos/Cargo.toml#L25 ?